app/Repositories/BaseRepository.php

```diff
@@ -0,0 +1,333 @@
+<?php
+namespace App\Repositories;
+
+use Illuminate\Support\Facades\DB;
+use App\Repositories\RepositoryInterface;
+
+abstract class BaseRepository implements RepositoryInterface
+{
+
+    /**
+     * @var $model
+     */
+    protected $model;
+    /**
+     * EloquentRepository constructor.
+     */
+    public function __construct()
+    {
+        $this->setModel();
+    }
+
+    /**
+     * set reponsitory instant
+     */
+    public function setModel()
+    {
+        $this->model = app()->make(
+            $this->getModel()
+        );
+    }
+
+    /**
+     * @param  int $id
+     * @return mixed
+     */
+    public function getById($id)
+    {
+        return $this->model->findOrFail($id);
+    }
+
+    /**
+     * @param  array $columns
+     * @return mixed
+     */
+    public function all($orderBy = 'updated_at desc', $columns = ['*'])
+    {
+        return $this->model->orderByRaw($orderBy)->get($columns);
+    }
+
+    /**
+     * @param  int   $perPage Per page
+     * @param  array $columns Array Columns
+     * @return array Return data paginate
+     */
+    public function paginate($perPage = 15, $columns = ['*'])
+    {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.FirstParamSpacing</br>**Severity**: error</br>**File**: app/Repositories/BaseRepository.php</br>**Line**: 241</br>The first parameter of a multi-line function declaration must be on the line after the opening bracket